services/movie_session.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+from datetime import datetime
+from db.models import MovieSession
+
+
+def create_movie_session(
+    movie_show_time: str, movie_id: int, cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        cinema_hall_id=cinema_hall_id,
+        movie_id=movie_id,
+    )
```

Comment:
The `create_movie_session` function does not validate the format of `movie_show_time`. Consider validating the input to ensure it is in the correct format before attempting to create a `MovieSession`.